Test/CoreSDK.Test/Shared/Extensibility/Implementation/PropertyTest.cs

```diff
@@ -291,6 +293,82 @@ public void SanitizeMeasurementsReplacesNegativeInfinityWith0()
             Assert.Equal(0, original["Key"]);
         }
 
+        [TestMethod]
+        public void SanitizeTelemetryContextTest()
+        {            
+            var telemetryContext = new TelemetryContext();
```

Comment:
nitpick: I would prefer to see the concrete type instead of var when the variable is not an anonymous type.